public/stylesheet.css

```
@@ -264,16 +264,27 @@ a {
     margin: 0 0 1rem;
   }
 }
-
 .header__list-item {
-  font-family: var(--font-family-stylized);
-  font-size: 1rem;
-  font-weight: 600;
-  margin: 0 0.7rem;
+  height: 55px;
+  width: 55px;
+  background-color: var(--color-background);
+  fill: var(--color-button-active);
 }
-.header__list-item:hover {
-  color: var(--color-button-hover);
-  text-decoration: underline;
+
+.header__list-item:hover,
+.header__list-icon:focus {
```

I feel like it would make more sense to use the same selector twice, what do you think @jorgeamadosoria?

```suggestion
.header__list-icon:hover,
.header__list-icon:focus {
```

---

it's not possible: header__list-item is a <li> element, it has no focus